ditto/src/main/java/eu/xenit/testing/ditto/api/AlfrescoBootstrapper.java

```diff
@@ -1,58 +1,110 @@
 package eu.xenit.testing.ditto.api;
 
+import eu.xenit.testing.ditto.api.data.ContentModel.Application;
 import eu.xenit.testing.ditto.api.data.ContentModel.Content;
 import eu.xenit.testing.ditto.api.data.ContentModel.System;
 import eu.xenit.testing.ditto.api.data.ContentModel.User;
+import eu.xenit.testing.ditto.api.model.Node;
+import java.util.function.Consumer;
 
-class AlfrescoBootstrapper {
+class AlfrescoBootstrapper<T extends DataSetBuilder> {
```

Comment:
Why exactly does this need to use a generic class?

Reply:
https://github.com/xenit-eu/ditto/pull/3/files/6d0bfe43a1428e735eed57db6c85a334bc82e9f3#diff-4f6df0a35555904d739044876a84d89bR20

I guess because here the return type of DefaultDataSetBuilder is desired

Reply:
because I'm using the exact type that is returned somewhere in the tests
maybe there are better solutions for this - this is in the public API, so I'd better make up my mind :-/